src/shared.c

```diff
@@ -1039,8 +1039,6 @@ char *GetUserInfo(const char *s, const char *key) {
 		}
 		s++;
 	}
-
-	return "";
```

Comment:
If I add a "break" up above, it will allow me to put this back without compiler yelling at me, so I'll do that.